charts/velero/crds/backups.yaml

```diff
@@ -1,11 +1,15 @@
+
+---
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
-  name: backups.velero.io
   labels:
     app.kubernetes.io/name: velero
   annotations:
-    "helm.sh/hook-delete-policy": "before-hook-creation"
```

Comment:
The hook-delete-policy is `before-hook-creation` ref to https://helm.sh/docs/topics/charts_hooks/#hook-deletion-policies. Therefore, we remove it.